django/db/models/functions/base.py

```diff
@@ -201,10 +201,7 @@ def __init__(self, expression, substring, **extra):
         expression: the name of a field, or an expression returning a string
         substring: a string to find inside expression
```

Comment:
`Substr.__init__()` does. I think it might provide some value. It looks like the other functions don't have parameters that need much explanation.